doc/releases/changelog-dev.md

```diff
@@ -284,6 +284,10 @@
   ```
   [(#4457)](https://github.com/PennyLaneAI/pennylane/pull/4457/)
 
+* `qml.gradients.pulse_generator` becomes `qml.gradients.pulse_odegen` to adhere to paper naming conventions. During v0.33, `pulse_generator`
+  is still available but raises a warning.
+  [(#4633)](https://github.com/PennyLaneAI/pennylane/pull/4633)
+
```

Comment:
@Qottmann please can you also add this to the [deprecations page](https://docs.pennylane.ai/en/stable/development/deprecations.html).

Reply:
Right! Thanks! ✅